crates/holochain/tests/tests/inline_zome_spec/mod.rs

```diff
@@ -143,36 +143,6 @@ async fn inline_zome_3_agents_2_dnas() -> anyhow::Result<()> {
     Ok(())
 }
 
-#[tokio::test(flavor = "multi_thread")]
```

Comment:
An unfinished ignored test that even the author cannot remember what it was for should just be deleted.

Reply:
> // I can't remember what this test was for? Should we just delete?

Haha, ffs! I'm sure there's something in V1 about it :)